src/main/java/seedu/address/model/person/JobTitle.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's job title in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidJobTitle(String)}
+ */
+public class JobTitle {
+
+    public static final String MESSAGE_CONSTRAINTS = "Job titles can take any values, and it should not be blank";
+
+    /*
+     * The first character of the job title must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[^\\s].*";
```

Comment:
For Job Titles (assuming job titles can only start with alphabets and contain only alphabets and certain special characters from thereon): ^[A-Za-z][A-Za-z_@.\#&+\-\s]*$
Can try it out, just a suggestion.